AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/Model/KeyStoreAdmin.smithy

```diff
@@ -194,7 +219,7 @@ structure VersionKeyInput {
 
   @required
   @documentation("Multi-Region or Single Region AWS KMS Key used to protect the Branch Key, but not aliases!")
```

Comment:
```suggestion
  @documentation("Multi-Region or Single Region AWS KMS Key ARNs used to protect the Branch Key")
```

Reply:
I am afraid an Alias can be an ARN.
It is just an ARN to an Alias, or an Alias ARN.

See [KMS' Developer Guide for Key identifiers](https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#key-id).

I will not that the Dev Guide agrees with you.
But I have been in MANY customer meetings where they assumed an alias would work
wherever a Key ARN is used.

Thus, I am very certain that we should continue to say "but not aliases!".

I am happy to add ARNs,
but as long as we do not support aliases,
we should be explicit about it.

We should NOT assume that customers have read the developer guide,
and realized that there are is a difference between a Key ARN and an Alias ARN.

Reply:
```suggestion
  @documentation("Multi-Region or Single Region AWS KMS Key ARN used to protect the Branch Key, but not aliases!")
```